ppl/archive/schema.go

```diff
@@ -135,9 +134,7 @@ func (ark *Archive) ReadDefinitions(ctx context.Context) (index.Definitions, err
 	return defs, err
 }
 
-type OpenOptions struct {
-	LogFilter []string
-}
+type OpenOptions struct{}
```

Comment:
I'm leaving this here as @mattnibs is about to add a field to it in #2020 .

Reply:
Much appreciated